packages/nextra/styles/steps.css

```diff
@@ -1,11 +1,16 @@
-.nextra-steps h3 {
-  counter-increment: var(--counter-id);
-
-  &:before {
-    @apply _absolute _size-[33px];
-    @apply _border-4 _border-[rgb(var(--nextra-bg))] _bg-gray-100 dark:_bg-neutral-800;
-    @apply _rounded-full _text-neutral-400 _text-base _font-normal _text-center _-indent-px;
-    @apply _mt-[3px] _ml-[-41px];
-    content: counter(var(--counter-id));
+.nextra-steps {
+  h2,
+  h3,
+  h4 {
+    counter-increment: var(--counter-id);
+    @apply _relative _mt-8 _border-none _text-xl _font-semibold;
```

Comment:
```suggestion
    @apply _border-none _p-0;
```
I think we can reset only border/padding styles from `nextra-theme-docs`, wdyt about 
<img width="809" alt="image" src="https://github.com/user-attachments/assets/06753af4-752f-4967-8f8c-b648a32d3a8f">